### ISSUE TITLE
Are any of these images from the original ImageNet validation set?

### ISSUE THREAD

**Issue by davidboxboro** [TACJu/PartImageNet#6]
Are any of these images from the original ImageNet validation set, or are they all from the training set?

**Comment by xjwu1024** [comment]
all imgs are from train set
